UnityGLTF/Assets/UnityGLTF/Tests/UnityGLTFTests.asmdef

```diff
@@ -1,12 +1,32 @@
 {
     "name": "UnityGLTFTests",
-    "optionalUnityReferences": [
-        "TestAssemblies"
-    ],
+    "rootNamespace": "",
     "references": [
         "UnityGLTFScripts",
-        "UnityGLTFEditor"
+        "UnityGLTFEditor",
+        "GLTFSerialization",
+        "UnityEngine.TestRunner",
+        "UnityEditor.TestRunner"
     ],
     "includePlatforms": [],
-    "excludePlatforms": []
+    "excludePlatforms": [],
+    "allowUnsafeCode": false,
+    "overrideReferences": true,
+    "precompiledReferences": [
+        "GLTFSerialization.dll",
+        "Newtonsoft.Json.dll",
+        "nunit.framework.dll"
+    ],
+    "autoReferenced": false,
+    "defineConstraints": [
+        "UNITY_INCLUDE_TESTS"
+    ],
+    "versionDefines": [
+        {
+            "name": "Select...",
```

Comment:
Mostly for my reference, but I'm unfamiliar with this `"Select..."` syntax! It normally points to a package name. What does this resolve to?

Reply:
It just means that a local package reference is missing and was never properly set up in the first place. Just kept for legacy reasons